core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -157,16 +156,27 @@ oppia.controller('FeedbackTab', [
           },
           newContent: function() {
             return $scope.activeThread.getReplacementHtmlFromSuggestion();
+          },
+          stateName: function() {
+            return $scope.activeThread.suggestion.stateName;
+          },
+          suggestionId: function() {
+            return $scope.activeThread.suggestion.suggestionId;
+          },
+          targetId: function() {
+            return $scope.activeThread.suggestion.targetId;
           }
         },
         controller: [
-          '$scope', '$log', '$uibModalInstance', 'suggestionIsHandled',
-          'suggestionIsValid', 'unsavedChangesExist', 'suggestionStatus',
-          'description', 'currentContent', 'newContent', 'EditabilityService',
+          '$scope', '$log', '$http', '$uibModalInstance', 'suggestionIsHandled',
+          'stateName', 'suggestionId', 'targetId', 'suggestionIsValid',
```

Comment:
please add these dependencies in sorted order.

Reply:
ohh i made mistake, sorii for that.